src/components/DrumPad.spec.js

```diff
@@ -0,0 +1,18 @@
+import DrumPad from './DrumPad.js';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+
+describe('DrumMachine', () => {
+  it('renders a section with sixteen buttons', () => {
+    const padClick = jest.fn()
+    render(<DrumPad handleClick={padClick} />);
+
+    const drumPad = screen.getByRole('button', { name: 'Drum Pad' });
+    expect(drumPad).toBeInTheDocument();
+
+    userEvent.click(drumPad)
```

Comment:
What do you want to test with it?
I would expect a second "it" here.

Reply:
It tests, if a click on the button calls the function which plays the Sound.
Yes a second it is good here 😅